libs/resiliency/include/hpx/resiliency/async_replay.hpp

```diff
@@ -142,6 +143,71 @@ namespace hpx { namespace resiliency { namespace experimental {
                     });
             }
 
+            template <std::size_t... Is>
+            hpx::future<Result> invoke_distributed(
+                hpx::naming::id_type id, hpx::util::index_pack<Is...>)
+            {
+                return hpx::async(f_, id, std::get<Is>(t_)...);
+            }
+
+            hpx::future<Result> call(std::vector<hpx::naming::id_type> ids,
```

Comment:
Should we use a distribution policy instead of the `vector<id_type>` here?

Reply:
I initially thought of keeping a distribution policy. The idea behind not keeping one is because the user will be interested in invoking them on particular ids. For standard actions, there isn't any difference between using a policy vs giving an id. For components, one can get the ids list when invoking new_ with distribution policy. So he can again invoke async with those ids.

Reply:
I'm open to suggestions though. So if you have something else in mind, I'd like to hear it out.

Reply:
I think we should follow the scheme we already have in place for `async` and friends. If there are additional use cases, those should be considered as a second step.